### DISCUSSION TITLE
Not Installable

### DISCUSSION THREAD

**Discussion by RewriteThis** [wp-cli/wp-cli#5726] [How do I?]
Running wp package install wp-cli/restful results in:

bash: fork: retry: No child processes
bash: fork: retry: No child processes
bash: fork: retry: No child processes
bash: fork: retry: No child processes
bash: fork: Resource temporarily unavailable

**Comment by danielbachhuber** [comment]
Hey @RewriteThis,

It's hard to say what the issue might be here. Feel free to stop by the `#cli` channel on [WordPress.org Slack](https://make.wordpress.org/chat/) if you'd like help debugging further.
